src/content/learn/describing-the-ui.md

```diff
@@ -358,11 +358,11 @@ export default function PackingList() {
 
 <LearnMore path="/learn/conditional-rendering">
 
-Baca **[Merender Secara Kondisional](/learn/conditional-rendering)** untuk mempelajari cara merender konten secara kondisional.
+Baca **[Me-render Secara Kondisional](/learn/conditional-rendering)** untuk mempelajari cara me-*render* konten secara kondisional.
 
 </LearnMore>
 
-## Merender List {/*rendering-lists*/}
+## Me-render List {/*rendering-lists*/}
```

Comment:
```suggestion
## Me-*render* List {/*rendering-lists*/}
```

Shall we make this *italic* as well?

Reply:
same reason at https://github.com/reactjs/id.react.dev/pull/503#discussion_r1181080945